.vscode/autogpt-workspace.code-workspace

```diff
@@ -0,0 +1,69 @@
+{
+  "folders": [
+    {
+      "name": ".github",
+      "path": "../.github"
+    },
+    {
+      "name": ".vscode",
+      "path": "../.vscode"
+    },
+    {
+      "name": "assets",
+      "path": "../assets"
+    },
+    {
+      "name": "autogpt",
+      "path": "../autogpt"
+    },
+    {
+      "name": "benchmark",
+      "path": "../benchmark"
+    },
+    {
+      "name": "docs",
+      "path": "../docs"
+    },
+    {
+      "name": "forge",
+      "path": "../forge"
+    },
+    {
+      "name": "frontend",
+      "path": "../frontend"
+    },
+    {
+      "name": "autogpt_server",
+      "path": "../rnd/autogpt_server"
+    },
+    {
+      "name": "root_folder",
+      "path": ".."
+    }
+  ],
+  "settings": {
+    "python.analysis.useLibraryCodeForTypes": true,
+    "python.analysis.autoImportCompletions": true,
+    "python.analysis.typeCheckingMode": "basic",
+    "python.analysis.completeFunctionParens": true,
+    "python.analysis.inlayHints.functionReturnTypes": true,
+    "python.analysis.inlayHints.variableTypes": true,
+    "python.analysis.autoFormatStrings": true,
+    "python.analysis.inlayHints.callArgumentNames": "partial",
```

Comment:
I have most of these features disabled because I don't like them.
```suggestion
    "python.analysis.inlayHints.functionReturnTypes": true,
    "python.analysis.autoFormatStrings": true,
```